### PR TITLE
Snes9x update

### PR DESCRIPTION
**Pull request by willco007**
Includes previous pull request plus Snes9x core update.

### CODE REVIEW

**Comment by jasarien** [comment]
Doesn't this pull request contain all the same content as your previous PR, with the addition of the SNES9x update?

If that is the case, you could close your previous PR and retain this one.

**Comment by willco007** [comment]
Yup, sure thing.

**Comment by willco007** [comment]
I've applied the changes from your comments on the FCEU PR as well as removed the background image border on the tvOS artwork as the OS applies it automatically.

File: Provenance.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -1247,7 +1247,7 @@
 					};
 					1AD481B31BA350A400FDA50A = {
 						CreatedOnToolsVersion = 7.1;
-						DevelopmentTeam = MF85NRPL4W;
+						DevelopmentTeam = 63497P68S6;
```
Comment (leolobato) [inline Provenance.xcodeproj/project.pbxproj]: It would be best to not change the development team.

Comment (willco007) [inline Provenance.xcodeproj/project.pbxproj]: Yes, this commit is to change it back to what it should be.

**Comment by jasarien** [comment]
It seems theres probably some issues with this SNES update: #417 Could you check them out @willco007 ?

**Pull request merged**